src/Sarif.Viewer.VisualStudio/Models/SarifErrorListItem.cs

```diff
@@ -32,6 +39,13 @@ public class SarifErrorListItem : NotifyPropertyChangedObject
         private ObservableCollection<XamlDoc.Inline> _messageInlines;
         private ResultTextMarker _lineMarker;
 
+        private static readonly Dictionary<FailureLevel, string> FailureLevelToPredefinedErrorTypes = new Dictionary<FailureLevel, string>
```

Comment:
Add a comment that describes this dictionary. This is indeed used to determine the color of the "squiggle" for a given SARIF result.